srml/session/src/lib.rs

```diff
@@ -316,6 +316,8 @@ decl_storage! {
 				let initial_validators = T::SelectInitialValidators::select_initial_validators()
 					.unwrap_or_else(|| config.keys.iter().map(|(ref v, _)| v.clone()).collect());
 
+				assert!(!initial_validators.is_empty(), "Empty validator set in genesis block!");
```

Comment:
This panics but I think it's fine since it's in `add_extra_genesis` (i.e. it's not part of block execution in runtime)